EIPS/eip-XXX-decentralize-EIP-workflow.md

```diff
@@ -0,0 +1,121 @@
+eip: <to be assigned>
```

Comment:
Please format this as per the instructions in EIP 0.

Reply:
> Please format this as per the instructions in EIP 0.

As per EIP 1.